actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+*   Raise exception when calling `to_h` in an unpermitted Parameters.
+
+    This method will raise on unpermitted Parameters if
+    `config.action_controller.raise_on_unfiltered_parameters` is true.
+
+    Before we returned either an empty hash or only the always permitted parameters
+    (`:controller` and `:action` by default).
+
+    The previous behavior was dangerous because in order to get the attributes users
+    usually fallback to use `to_unsafe_h that` could potentially introduce security issues.
+
+    *Rafael Mendonça França*
+
+*   Add `ActionController::Parameters#to_hash` to implicit conversion.
+
+    Now methods that implicit convert objects to a hash will be able to work without
+    requiring the users to change their implementation.
+
+    This method will return a `Hash` instead of a `HashWithIndefirentAccess` to mimic the
+    same implementation of `HashWithIndefirentAccess#to_hash`.
```

Comment:
Why don't we expand this to `ActiveSupport::HashWithIndifferentAccess` since Action Controller isn't the namespace here?